nucypher/characters/lawful.py

```diff
@@ -1008,6 +1012,9 @@ def __init__(self,
                                                    rest_app=rest_app, datastore=datastore,
                                                    hosting_power=tls_hosting_power)
 
+                self._arrangement_pruning_task = LoopingCall(f=self.__prune_arrangements)
+                self._arrangement_pruning_task.start(interval=self._pruning_interval)  # TODO: Move to ursula.run ?
```

Comment:
What about moving this to the WorkTracker? I can see it being related.

Reply:
`WorkTracker` is only implemented for non-federated operation.